datacommons/populations.py

```diff
@@ -88,34 +80,22 @@ def get_populations(dcids, population_type, constraining_properties={}):
       "geoId/21": "dc/p/fs929fynprzs",
       "geoId/24": "dc/p/lr52m1yr46r44"
     }
-
-    We can also specify the :code:`dcids` as a :obj:`pandas.Series` like so.
-
-    >>> import pandas as pd
-    >>> dcids = pd.Series(["geoId/06", "geoId/21", "geoId/24"])
-    >>> pvs = {'employment': 'BLS_Employed'}
-    >>> dc.get_populations(dcids, 'Person', constraining_properties=pvs)
-    0    dc/p/x6t44d8jd95rd
-    1     dc/p/fs929fynprzs
-    2    dc/p/lr52m1yr46r44
-    dtype: object
   """
   # Convert the dcids field and format the request to GetPopulations
-  dcids, req_dcids = utils._convert_dcids_type(dcids)
+  dcids = list(dcids)
   pv = [{'property': k, 'value': v} for k, v in constraining_properties.items()]
   url = utils._API_ROOT + utils._API_ENDPOINTS['get_populations']
   payload = utils._send_request(url, req_json={
-    'dcids': req_dcids,
+    'dcids': dcids,
     'population_type': population_type,
     'pvs': pv,
   })
 
   # Create the results and format it appropriately
   result = utils._format_expand_payload(
     payload, 'population', must_exist=dcids)
-  if isinstance(dcids, pd.Series):
-    flattened = utils._flatten_results(result, default_value="")
-    return pd.Series([flattened[dcid] for dcid in dcids], index=dcids.index)
+  print(result)
```

Comment:
Should these print statements be removed?

Reply:
Done